cmd/kubeapps-apis/plugins/kapp_controller/packages/v1alpha1/server_test.go

```diff
@@ -66,16 +70,19 @@ func TestGetClient(t *testing.T) {
 		t.Run(tc.name, func(t *testing.T) {
 			s := Server{clientGetter: tc.clientGetter}
 
-			client, err := s.GetClient(context.Background())
+			typedClient, dynamicClient, err := s.GetClients(context.Background())
 
 			if got, want := status.Code(err), tc.statusCode; got != want {
 				t.Errorf("got: %+v, want: %+v", got, want)
 			}
 
-			// If there is no error, the client should be a dynamic.Interface implementation.
+			// If there is no error, the clients should not be nil.
 			if tc.statusCode == codes.OK {
-				if _, ok := client.(dynamic.Interface); !ok {
```

Comment:
Not sure why I initially used a type conversion here, since if it's non-nil, it's going to be the correct type (or it wouldn't compile).